### ISSUE TITLE
Tab index not possible to set for Tooltip or DatePicker

### ISSUE THREAD

**Issue by vron** [dnbexperience/eufemia#3287]
It is currently not possible to specify tab-index on neither the Tooltip nor the DatePicker component - meaning that when I have a DatePicker Wrapped in a Tooltip and the suer is navigating with the keybord the "same" visible UI element will be selected twice.

I would expect both Tooltip *and* DatePicker to support the tab-index propery such that it is propagated to the correct html elements when rendered.

I have tried various work-arounds but not managed to work around it

**Comment by tujoworker** [comment]
Thanks for sharing your request ⭐

I think, this is also a question about UX, so maybe you can share more details about your case, so we could collaborate about possible other solutions?

**Comment by vron** [comment]
Hi  @tujoworker and thanks,

So the usecase is that we have a tool where the user needs to (quickly) input a lot of data - among them several dates.

To make the usability better we have developed a custom date-parser (such that it can support expressions such as "5m", "2y", "1y+2m" etc.). However - a few user prefer to "point and click" and we thus provide a few "quick buttons" and a "calendar drop down" for them.

for the common scenario though I want to allow "tab" to take them straight to the next input and not have to press tab 3 times (one moves selection to tooltip, next to the calendar button and the thrird to the next input) to move to the next field.

Maybe a screenshot makes this clearer:


![image](https://github.com/dnbexperience/eufemia/assets/160233/25737991-aa6e-4402-ae6f-a05ecc6c96fc)


(Note that the Tooltip is used to provide visual feedback when the user want it that the calculated date from the expression is correct)

**Comment by vron** [comment]
If you had any input / suggestions for what I can do to mitigate this it would be apprechiated :-)

**Comment by tujoworker** [comment]
Not really. Because using tab is intuitive. Else we could have added custom keyboard handling to jump directly to the input. E.g. `ArrawDown` and `ArrowUp`. 

For sure, forbidding tabbing on certain elements should not be done for external users. But I think its an internal app, right?

Added a PR to support both tooltip and tabIndex. This way you can provide the tooltip content directly to the date picker.

**Comment by vron** [comment]
Thanks, yes this is indeed for internal purposes. (But as an aide to educate me - does accessibility requirements require that all elements are reacheable or only all funcitonality? E.g. in a case such as this where one can tab to the date input and type it - is that sufficient or must all alternative ways of input be reacheable? - guessing that was the concern if it was an external app?)

**Comment by tujoworker** [comment]
Yes, it would be a concern we would have to taken care of.
